first try at listing OT training schedule (#42)
uwhackweek/hackweeks-as-a-service: docs/resources/index.md

# Resources for Organizers

> The eScience Institute supports Community Partners in planning each hackweek. We invite organizers to participate in video and live training sessions tailored to each of the Community Partner roles. Upon completion of the training, organizer profiles will be posted on the hackweek website along with badge(s) representing their roles in the event. 

Reviewer: Should we add a link to the hackweek website so people can see where they'll be posted?

::::{grid}
:gutter: 1

> :::{grid-item-card} [Tutorial Lead](#tutorial-lead)

Reviewer: Currently these links go to the meeting schedule info on this page - should we have the title or badge link back to the actual role page instead?

Author: I saw these as navigation tools within the page, and wonder if it will be confusing if they link back to the roles page?

For now I'll suggest we have an option to link back to the roles within the text describing the schedules. We can revisit this later if it doesn't seem to flow well.

> :::{grid-item-card} [Tutorial Lead](#tutorial-lead)

Reviewer: ```suggestion
:::{grid-item-card} [Tutorial Developer](#tutorial-developer)
```

> [![GH](https://img.shields.io/static/v1?label=eScience&message=Tutorial-Lead&color=b7a57a)](#tutorial-lead)

Reviewer: ```suggestion
[![GH](https://img.shields.io/static/v1?label=eScience&message=Tutorial-Developer&color=b7a57a)](#tutorial-developer)
```

:::

:::{grid-item-card} [Project Coordinator](#project-coordinator)
[![GH](https://img.shields.io/static/v1?label=eScience&message=Project-Coordinator&color=85754d)](#project-coordinator)
:::

:::{grid-item-card} [Helper](#helper)
[![GH](https://img.shields.io/static/v1?label=eScience&message=Helper&color=4b2e83)](#helper)
:::

::::

> ## Tutorial Lead Meeting Schedule

Reviewer: ```suggestion
## Tutorial Developer Meeting Schedule
```

> Tutorial leads and developers will gather two months prior to a hackweek to create content and prepare for teaching. Here is the weekly schedule:

Reviewer: ```suggestion
Tutorial leads and developers typically gather over the course of eight to ten weeks prior to a hackweek to create content and prepare for teaching. Below is an example eight week meeting schedule:
```

```{Admonition} Week 1 / 90 min: Kick-off Meeting (everyone)
Create tutorial schedule outline, form tutorial teams, and get to know each other.
```

```{Admonition} Week 2: Video Training (individual)
:class: tip
**Tutorial Development (20 min)**: Best practices, pedagogy, learning outcomes, and scoping a tutorial.

**Technical Infrastructure (20 min)**: Overview of Jupyter Notebooks and pushing content to our Jupyter Book.
```

```{Admonition} Week 2 (60 min): Tutorial Outlining Meeting (tutorial teams)
Meet with tutorial team to scope and outline tutorial content.
```

```{Admonition} Week 3 (60 min): Tutorial Report-Out Meeting (everyone)
Each team shares scope and outline and together we identify overlaps, ways to integrate content and decide on tutorial sequencing.
```

```{Admonition} Week 3: Open Office Hours (individual)
:class: tip

> Get help with technical infrastructure and practice doing a pull request to the Jupyterbook.

Reviewer: ```suggestion
Get help with technical infrastructure and practice doing a pull request to the Jupyter Book.
```

```

```{Admonition} Week 4: Video Training (individual)
:class: tip
**Tutorial Development (20 min)**: Learn ways to engage with participants and incorporate peer-learning into your lessons.
```

```{Admonition} Week 4 (60 min): Tutorial Development (tutorial teams)
Discuss places/ways to engage participants and incorporate interactive components.
```

```{Admonition} Week 5: Open Office Hours (individual)
:class: tip
Tutorial development help.
```

```{Admonition} Week 6 and 7 (60 min): Tutorial Feedback (everyone)

> Practice tutorial delivery and get constructive feedback from others on the team.

Reviewer: ```suggestion
Practice tutorial delivery and get constructive feedback from others on the organizing team.
```

Trying to make sure our use of "team" is clear whether we mean an individual tutorial team, the whole tutorial and education team, or the whole organizing team.

```

```{Admonition} Week 8 (60 min): Final meeting (everyone)
Coordination, logistics, and final preparations.
```

> Upon completion of this training you will be awarded a tutorial lead badge! ![GH](https://img.shields.io/static/v1?label=eScience&message=Tutorial-Lead&color=b7a57a).

Reviewer: ```suggestion
Upon completion of this training you will be awarded a tutorial developer or lead badge! ![GH](https://img.shields.io/static/v1?label=eScience&message=Tutorial-Developer&color=b7a57a)
![GH](https://img.shields.io/static/v1?label=eScience&message=Tutorial-Lead&color=b7a57a)
```

> ## Project Coordinator

Reviewer: ```suggestion
## Project Coordinator Preparation Schedule

The project coordinator's tasks begin four to six weeks prior to a hackweek, after participant selection is completed. A more detailed schedule is coming soon!
```

> # Helper

Reviewer 2: ```suggestion
## Helper
```

> ## Helper

Reviewer: ```suggestion
## Helper

Helpers are critical to keeping a hackweek running smoothly and sharing information among participants and organizers. Helpers are asked to attend a virtual training session (60 minutes), usually the week before the hackweek, to get an overview of the event, have an opportunity to ask questions, and commit to being a helper for a specific day.
```
